sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/misc-files-0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 The Flatcar Maintainers.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION='Flatcar bashrc files'
+HOMEPAGE='https://www.flatcar.org/'
+
+LICENSE='Apache-2.0'
+SLOT='0'
+KEYWORDS='amd64 arm64'
+
+# No source directory.
+S="${WORKDIR}"
+
+# Versions listed below are version of packages that shedded the
+# modifications in their ebuilds.
+RDEPEND="
+	>=app-shells/bash-5.2_p15-r2
+"
+
+# Use absolute paths to be clear about what locations are used. The
```

Comment:
We also have similar things in `coreos-base/coreos-init`. Maybe we could put everything there instead of creating a new package, but as you like.

Reply:
It's a bit of a mess, you know - all this compat symlinks stuff is spread over several places (baselayout Makefile, baselayout ebuild, init Makefile, init ebuild, probably more). I'll have a look where I could add those instead of creating yet another place where we install random files and symlinks.

Reply:
I decided to keep the `coreos-base/misc-files` package, as `coreos-base/coreos-init` should be about systemd stuff like units. Compat symlinks and other extra files could go to the misc-files package. I think it will evolve as we will keep moving more and more packages from overlay back to portage-stable.